src/Library/demos/GTK Switch/main.blp

```diff
@@ -0,0 +1,55 @@
+using Gtk 4.0;
+using Adw 1;
+
+Adw.StatusPage {
+  title: "GTK Switch";
+
+  Box {
+    orientation: vertical;
+
+     Label {
+        label: _("GtkSwitch is a “light switch” that has two states: on or off.\nThe user can control which state should be active by clicking the empty area, or by dragging the handle.\n");
+        justify: center;
+        margin-bottom: 15;
+        styles ["body"]
+      }
+
+    Box {
+      orientation: vertical;
+      halign: center;
+
+      Switch one{
+      margin-bottom: 10;
+      active: true;
+      }
+    }
+    Label switch_one{
+    label: "Currently in Active state";
+    margin-bottom: 40;
+    }
+
+    Box {
+     orientation: vertical;
+     halign: center;
+
+     Switch second{
+     margin-bottom: 10;
+     }
+    }
+
+    Label switch_second{
+    label: "Currently in Inactive state";
+    margin-bottom: 20;
+    }
```

Comment:
Below the second switch – can you add a disabled switch? (`sensitive: false`) with the label "Disabled"